redash/cli/rq.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+from datetime import datetime, timedelta
+from random import randint
+
+from redash import redis_connection, settings
+from redash.tasks import (sync_user_details, refresh_queries,
+                          empty_schedules, refresh_schemas,
+                          cleanup_query_results,
+                          version_check, send_aggregated_errors)
+
+from click import argument
+from flask.cli import AppGroup
+from rq import Connection, Worker
+from rq_scheduler import Scheduler
+
+manager = AppGroup(help="RQ management commands.")
+
+
+@manager.command()
+def scheduler():
+    scheduler = Scheduler(connection=redis_connection,
+                          queue_name="periodic",
+                          interval=5)
```

Comment:
Let's move this (along with the task definitions) to `redash.worker` and have the scheduler object defined at the module level.

Reply:
Eventually went for `redash.schedule` to (a) avoid bloat (b) avoid circular dependencies (c) worker/scheduler is a mesh-up which originated in Celery and is represented by separate concepts in RQ.